EDU-12223 - Update Headless CMS widgets doc (#1329)
vtexdocs/dev-portal-content: docs/faststore/docs/headless-cms/widgets.mdx

              },
              "icon": {
                "title": "Icon",
                "type": "string",
                "enumNames": ["CircleWavyCheck"],
                "enum": ["CircleWavyCheck"],
                "default": "CircleWavyCheck"
              }
            }
          },
          "toastSubscribeError": {
            "title": "Toast Subscribe Error",
            "type": "object",
            "properties": {
              "title": {
                "title": "Title",
                "description": "Message Title",
                "type": "string",
                "default": "Oops."
              },
              "message": {
                "title": "Message",
                "description": "Message",
                "type": "string",
                "default": "Something went wrong. Please try again."
              },
              "icon": {
                "title": "Icon",
                "type": "string",
                "enumNames": ["CircleWavyWarning"],
                "enum": ["CircleWavyWarning"],
                "default": "CircleWavyWarning"
              }
            }
          }
        }
      }
    },

  …

  ```

  </CH.Code>

>   </CH.Scrollycoding>

Reviewer: ```suggestion
```

### Synchronize the changes with Headless CMS

4. Open the terminal and run `yarn cms-sync` to synchronize your changes with the Headless CMS. Once the changes are synchronized, you will see the following message: `info: CMS synced successfully...`.
5. Go to the VTEX Admin and access **Storefront > Headless CMS**.
6. Open the page where you want to add the new section. In this context, we will add the `Custom Newsletter` to the `Home`.
7. Add the section to the page, and you will notice the widget you have added, the [`media-gallery`](#media-gallery).

    ![newsletter-hcms-interface](https://vtexhelp.vtexassets.com/assets/docs/src/scoped-widgets-one___6c8c5e28b43487efc6f6048afd8d8d15.gif)

8. Click `Preview` to see the new component with the new image.

>     ![newsletter-store-interface](https://vtexhelp.vtexassets.com/assets/docs/src/scoped-widgets-two___9c76a69ce03639b33faa85eb2a7cb367.png)

Reviewer: ```suggestion
    ![newsletter-store-interface](https://vtexhelp.vtexassets.com/assets/docs/src/scoped-widgets-two___9c76a69ce03639b33faa85eb2a7cb367.png)

</CH.Scrollycoding>
```

    "widget": {
       "ui:widget": "media-gallery"
     }
 },
```
</CH.Code>

---

## Adding a widget to a store section

Widgets can help when you have [created a new store section](https://developers.vtex.com/docs/guides/faststore/building-sections-creating-a-new-section) and want to add a new property to it. To illustrate how a widget can be used in a store section, below there’s an example using the [`media-gallery`](#media-gallery) widget in the new `Custom Newsletter` section based on the [`Newsletter`](https://developers.vtex.com/docs/guides/faststore/organisms-newsletter) native section.

> ℹ️ To create a new section, such as the `Custom Newsletter`, follow the instructions in the [Creating a new store section](https://developers.vtex.com/docs/guides/faststore/building-sections-creating-a-new-section) guide.

1. Open your store project code in a code editor and navigate to the `cms/faststore` folder.
2. Open the `sections.json` file.
3. In the `sections.json` file, find the section where you want to add the widget. In the following `sections.json` file example, the new section, `Custom Newsletter`, receives the [`media-gallery`](#media-gallery) widget.

>   <CH.Code>
> 
>   ```json sections.json mark[34:39]

Reviewer 2: Ajustando indentação

```suggestion
<CH.Code>

```json sections.json mark[34:39]
```

              },
              "icon": {
                "title": "Icon",
                "type": "string",
                "enumNames": ["CircleWavyCheck"],
                "enum": ["CircleWavyCheck"],
                "default": "CircleWavyCheck"
              }
            }
          },
          "toastSubscribeError": {
            "title": "Toast Subscribe Error",
            "type": "object",
            "properties": {
              "title": {
                "title": "Title",
                "description": "Message Title",
                "type": "string",
                "default": "Oops."
              },
              "message": {
                "title": "Message",
                "description": "Message",
                "type": "string",
                "default": "Something went wrong. Please try again."
              },
              "icon": {
                "title": "Icon",
                "type": "string",
                "enumNames": ["CircleWavyWarning"],
                "enum": ["CircleWavyWarning"],
                "default": "CircleWavyWarning"
              }
            }
          }
        }
      }
    },

  …

  ```

>   </CH.Code>

Reviewer 2: Ajustando identação. Lembrar de ajustar o fechamento do bloco de código (` ``` `) acima.

```suggestion
</CH.Code>
```

> 

Author: ```suggestion

---
```

### Synchronize the changes with Headless CMS

> 4. Open the terminal and run `yarn cms-sync` to synchronize your changes with the Headless CMS. Once the changes are synchronized, you will see the following message: `info: CMS synced successfully...`.
> 5. Go to the VTEX Admin and access **Storefront > Headless CMS**.
> 6. Open the page where you want to add the new section. In this context, we will add the `Custom Newsletter` to the `Home`.
> 7. Add the section to the page, and you will notice the widget you have added, the [`media-gallery`](#media-gallery).
> 
>     ![newsletter-hcms-interface](https://vtexhelp.vtexassets.com/assets/docs/src/scoped-widgets-one___6c8c5e28b43487efc6f6048afd8d8d15.gif)
> 
> 8. Click `Preview` to see the new component with the new image.

Reviewer 2: Os números da lista aqui estão corretos? Parece que estão continuando a contagem da lista da seção anterior, e não sei se deveria ser assim. Vou deixar a sugestão aqui. Outra opção seria remover o cabeçalho, e talvez ajustar o texto do item 4.

```suggestion
1. Open the terminal and run `yarn cms-sync` to synchronize your changes with the Headless CMS. Once the changes are synchronized, you will see the following message: `info: CMS synced successfully...`.
2. Go to the VTEX Admin and access **Storefront > Headless CMS**.
3. Open the page where you want to add the new section. In this context, we will add the `Custom Newsletter` to the `Home`.
4. Add the section to the page, and you will notice the widget you have added, the [`media-gallery`](#media-gallery).

    ![newsletter-hcms-interface](https://vtexhelp.vtexassets.com/assets/docs/src/scoped-widgets-one___6c8c5e28b43487efc6f6048afd8d8d15.gif)

5. Click `Preview` to see the new component with the new image.
```
